app/views/dashboard/_links.njk

```diff
@@ -63,17 +63,16 @@
         <article class="{{columnClass}} links__box" id="stripe-test-account-requested">
           <h2 class="govuk-heading-s govuk-!-margin-bottom-2">Stripe test account requested</h2>
           <p class="govuk-body govuk-!-margin-bottom-0">
-            You've requested a stripe account from the GOV.UK Pay team.
-            We're setting up your test account. It will be ready within 2 working days.
+            You’ve requested a stripe account from the GOV.UK Pay team.
+            We’re setting up your test account. It will be ready within 2 working days.
           </p>
         </article>
       {% else %}
         <article class="{{columnClass}} links__box" id="request-stripe-test-account">
           <a href="/service/{{ currentService.externalId }}/request-stripe-test-account" class="govuk-link">
-            <h2 class="govuk-heading-s govuk-!-margin-bottom-2">Request Stripe test account</h2>
-            <p class="govuk-body govuk-!-margin-bottom-0">Use GOV.UK’s Payment Service Provider (PSP) to view transaction fee from each payment.
-             Set up a Stripe account to test payments, reporting, and reconciliation.
-            </p>
+            <h2 class="govuk-heading-s govuk-!-margin-bottom-2">Request stripe test account</h2>
```

Comment:
Does the content doc say that the tile should now be `Test Stripe reporting functionality` instead of `Request stripe test account`?

Reply:
Good spot! updated